src/System.Net.Security/tests/Scripts/setup-kdc.sh

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/env bash
+
+OS=`cat /etc/os-release | grep "^ID=" | sed 's/ID=//g' | sed 's/["]//g' | awk '{print $1}'`
+echo -e "Operating System: ${OS}\n"
+
+realm="TEST.COREFX.NET"
+
+principal1="TESTHOST/testfqdn.test.corefx.net"
+principal2="TESTHTTP"
+krb_user="krb_user"
+password="password"
+
+kadmin="kadmin.local"
+krb5kdc="krb5kdc"
+kdb5_util="kdb5_util"
+add_principal_cmd="add_principal -pw ${password}"
+
+krb_conf="krb5.conf"
+krb_conf_location="/etc/krb5.conf"
+keytabfile="/etc/krb5.keytab"
+
+PROGNAME=$(basename $0)
+usage()
+{
+	echo "This script must be run with super-user privileges."
+	echo "Usage: ${PROGNAME} [-h|--help] [-y|--yes] [-u|--uninstall]";
+}
+
+# Cleanup config files and uninstall KDC
+clean_up()
+{
+	echo "Stopping KDC.."
+	if pgrep krb5kdc 2> /dev/null; then killall krb5kdc ; fi
+	
+	echo "Removing config files"
+	if [ -f ${krb_conf_location} ]; then
+		rm -f ${krb_conf_location}
+	fi
+	
+	case ${OS} in
+		"ubuntu" | "debian")
+			kdc_conf_location="/etc/krb5kdc/kdc.conf"
+			dpkg -s krb5-kdc >/dev/null 2>&1
+			if [ $? -eq 0 ]; then
+				echo "Uninstalling krb5-kdc"
+				apt-get -y purge krb5-kdc
```

Comment:
Uninstall is done via passing "--uninstall".
 Moreover, the test fixture will install it if needed and uninstall it at the end of the test run.